app/models/user.rb

```diff
@@ -1,9 +1,15 @@
 class User < ApplicationRecord
+# RELATIONSHIPS
   has_many :user_games
   has_many :games, through: :user_games
   has_many :friendships
   has_many :friends, through: :friendships, source: :friend
 
   has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
   has_many :inverse_friends, through: :inverse_friendships, source: :user
+
+# VALIDATIONS
```

Comment:
Nice work - Im assuming that having a condition as the value for key uniqueness means that uniqueness is "true" AND "case_sensitive:false".